content/docs/started/getting-started.md

```diff
@@ -26,11 +26,11 @@ functionality and actively soliciting help from companies and individuals intere
 
 ## Setup
 
-This documentation assumes you have a Kubernetes cluster already available.
+This documentation assumes you have a Kubernetes cluster available. If
+not, setup one of these environments first:
 
-  * For local environment try [Minikube setup](/docs/started/getting-started-minikube/).
-  * For cloud environment try [GKE setup](/docs/started/getting-started-gke/).
-  * For general k8s cluster try [quick start](/docs/started/getting-started/#quick-start) or [install via ksonnet](/docs/started/getting-started/#install-via-ksonnet).
+  * Local: [Minikube setup](/docs/started/getting-started-minikube/)
+  * Cloud: [GKE setup](/docs/started/getting-started-gke/)
```

Comment:
I think we need a 3rd option directing people on what to do if they have an existing cluster that's not minikube/GKE

Reply:
The sentence before these two bullet items assumes an existing k8s cluster. The purpose of these two links is to give people an option if that's not the case.